src/collection/types.ts

```diff
@@ -0,0 +1,44 @@
+import {
+	BlockImageProps,
+	BlockImageTitleTextButtonProps,
+	BlockIntroProps,
+	BlockLinksProps,
+	BlockQuoteProps,
+	BlockSubtitleProps,
+	BlockTextProps,
+	BlockTitleImageTextProps,
+	BlockTitleProps,
+	BlockVideoProps,
+	BlockVideoTitleTextButtonProps,
+} from '@viaa/avo2-components';
+
+export type ContentBlockType =
```

Comment:
Je kan van deze type een enum maken. Zo kan je die ook exporten en gebruiken in andere files ipv met strings te moeten werken.